### DISCUSSION TITLE
Improve the navigation on the right sidebar

### DISCUSSION THREAD

**Discussion by juniusfree** [athensresearch/athens#1008] [Feature Requests]
Problem: It gets harder/slower to navigate the right sidebar the more you add pages to it.

Some ideas:
1. Collapse All pages
2. Billet up/down through keyboard command https://discord.com/channels/708122962422792194/800439780910563358/834418132751613982
2.1 like the move up/down line in vscode https://discord.com/channels/708122962422792194/800439780910563358/834423431893745664

**Comment by shanberg** [comment]
Is this a matter of performance or just the time it takes to scroll?

**Reply by juniusfree** [reply]
Time to find the page that I added on the right sidebar.

**Reply by juniusfree** [reply]
> The sidebar is a good solution to this problem for a stack size of one. I'd love to see good approaches for larger stack sizes and non-stack-ish browsing patterns...
https://twitter.com/andy_matuschak/status/1240841416642686977

**Reply by juniusfree** [reply]
https://discord.com/channels/708122962422792194/800439780910563358/834663454555111454
